From cf7362afe670706a385fdba579deef4ad455aba7 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Fri, 5 Feb 2016 12:10:56 +0100
Subject: [PATCH] add I$JAVA_HOME/include/linux to JAVAINC to kFreeBSD to fix
 build

jni_md.h not in $JAVA_HOME/include anymore in OpenJDK8 (it was in 7)

Change-Id: Id5c737182a8e476b609887753047fa7ed6b01193
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 3175689..fec39bf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -7179,6 +7179,7 @@ if test -n "$ENABLE_JAVA" -a -z "$JAVAINC"; then
 
     k*bsd*-gnu*)
         JAVAINC="-I$JAVA_HOME/include"
+        JAVAINC="$JAVAINC -I$JAVA_HOME/include/linux"
         test -d "$JAVA_HOME/include/native_thread" && JAVAINC="$JAVAINC -I$JAVA_HOME/include/native_thread"
         ;;
 
-- 
2.1.4

